### PR TITLE
[WIP] moving my improvements upstream

### PR DESCRIPTION
**Pull request by maurelio1234**
The intent is not merging this PR as is, but using it as a big todo list.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
-    cronStarted = true;
+// });
 
-}
\ No newline at end of file
+// cronStarted = true;
+
+// }
+exports.handler = (event, context, callback) => {
```
Comment (maurelio1234) [inline index.js:18]: make it compatible with the heroku index.

Maybe we just need to export the handler and disable the cron via env vars

File: services/goquest_service.js [hunk 2]
```diff
@@ -7,150 +7,169 @@ const ogs = require('../services/ogs_service');
...
 
-                    console.log("uploading sgf for game " + GoQuestGame.getFileName(goQuestGame));
+        console.log("received game", message);
 
-                    try {
-
-                        let sgf = GoQuestGame.toSgf(goQuestGame.args[0].players, goQuestGame.args[0].position, goQuestGame.args[0].gtype);
-                        await ogs.uploadSgf(GoQuestGame.getFileName(goQuestGame), sgf);
+        let goQuestGame = message;
+        if (goQuestGame.args[0].error)
```
Comment (maurelio1234) [inline services/goquest_service.js:56]: already migrated

File: services/goquest_service.js [hunk 3]
```diff
@@ -7,150 +7,169 @@ const ogs = require('../services/ogs_service');
...
 function getGame(gameId) {
-
-    console.log("getting game");
-    let gameMessage = process.env.GQ_GAME_MESSAGE.replace("GAME_ID", gameId);
-    console.log("sending message");
-    console.log(gameMessage);
-    ws.send(gameMessage);
-
+  console.log("getting game");
+  let gameMessage = process.env.GQ_GAME_MESSAGE.replace("GAME_ID", gameId).replace("PROFILE_ID", process.env.GQ_PROFILE_NAME);
```
Comment (maurelio1234) [inline services/goquest_service.js:132]: TODO: create issue: to improve configuraton of GQ_GAME_MESSAGE

**Comment by andrewallred** [comment]
> The intent is not merging this PR as is, but using it as a big todo list.

Ok, I'll take a look this evening. Thanks for putting this together!

File: services/goquest_service.js (hunk 3 above)
Comment (maurelio1234) [reply services/goquest_service.js:132]: #10

**Pull request closed without merging**